src/topics/group-generator/group-generator.test.ts

```diff
@@ -458,13 +486,16 @@ describe("test group generator", () => {
       },
     });
     const groups = await groupStore.all();
-    expect(Object.keys(groups)).toHaveLength(1);
+    expect(Object.keys(groups)).toHaveLength(2);
     const data = await Object.values(groups)[0].data();
     expect(data["0x0000000000000000000000000000000000000030"]).toBe("1");
     expect(data["0x0000000000000000000000000000000000000031"]).toBe("2");
+    const data2 = await Object.values(groups)[1].data();
```

Comment:
Maybe also check the value of one key per group that is different from 0x...30 and 0x...31 to be sure to have generated different groups?